cmd/cleanup/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/sig-storage-local-static-provisioner/pkg/cleanup/controller"
+	"sigs.k8s.io/sig-storage-local-static-provisioner/pkg/cleanup/deleter"
+)
+
+// Command line flags
+var (
+	kubeconfig               = flag.String("kubeconfig", "", "Absolute path to the kubeconfig file. Only required when running out of cluster.")
+	masterURL                = flag.String("master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+	resync                   = flag.Duration("resync", 10*time.Minute, "Resync interval of the controller.")
+	storageClassName         = flag.String("storageClassName", "", "Name of StorageClass to opt-in PVs and PVCs for cleanup.")
```

Comment:
Could this be a list?

Reply:
Changed to list of storageclass names such that if PV/PVC belong to any one of the names in the list, they are eligible for cleanup.